CHANGELOG.md

```diff
@@ -1,13 +1,16 @@
 # Changelog
 
-## 3.10.0
+## Unreleased
 
 ### Features
 
+- Add OkHttpEventListener automatically ([#504](https://github.com/getsentry/sentry-android-gradle-plugin/pull/504))
 - New Sqlite instrumentation ([#502](https://github.com/getsentry/sentry-android-gradle-plugin/pull/502))
     - This integration replaces the old database instrumentation with the `sentry-android-sqlite` integration
     - Any implementation of SupportSQLiteOpenHelper.Factory is now supported
 
+## 3.10.0
```

Comment:
looks like the new db instrumentation was not added in 3.10.0 (i downloaded the [source](https://github.com/getsentry/sentry-android-gradle-plugin/releases/tag/3.10.0) and checked it)
Probably it was just a wrong changelog update after a merge with master

Reply:
Oh my bad then, I assumed the one on main branch was correct. Thanks for fixing 👍

Reply:
yeah, it stealthily slipped through to main when i merged the other pr 😅